corehq/apps/analytics/tasks.py

```diff
@@ -148,6 +148,13 @@ def _send_form_to_hubspot(form_id, webuser, cookies, meta):
 
 
 @task(queue='background_queue', acks_late=True, ignore_result=True)
+def update_hubspot_properties(webuser, properties):
+    vid = _get_user_hubspot_id(webuser)
+    if vid:
+        _track_on_hubspot(webuser, properties)
```

Comment:
This is hitting hubspot twice: 

- `https://api.hubapi.com/contacts/v1/contact/email/{email}/profile` to verify account exists
- `https://api.hubapi.com/contacts/v1/contact/createOrUpdate/email/{email}` to track

Does hubspot have an API to update an existing account but not create a new one? I assume that's what we want since that appears to be what's happening here.